### ISSUE TITLE
fix: Consistent databaseId

### ISSUE THREAD

**Issue by moonmeister** [wp-graphql/wp-graphql#1166]
Posts, pages, and other things have dpreciated the per type `postId` to access the WP Id in favor of `databaseID` to be consistent across types. This is Good but not all types have gotten this treatment and still need replacement/depreciation:

- menuId
- menuItemId
- there may be more I haven't noticed...

**Comment by kellenmace** [comment]
Also –
Although the `databaseId` has been added for users, the `userId` field has not yet been deprecated. I assume we want to deprecate it, since we've deprecated the `postId` field for posts.

**Comment by jasonbahl** [comment]
Closed by #1348 

For example:

```graphql
{
  menus(first:1) {
    edges {
      cursor
      node {
        id
        name
        databaseId
        menuItems {
          nodes {
            id
            databaseId
            menu {
              node {
                name
              }
            }
          }
        }
      }
    }
  }
}
```

Returns the following:

```json
{
  "data": {
    "menus": {
      "edges": [
        {
          "cursor": "YXJyYXljb25uZWN0aW9uOjUz",
          "node": {
            "id": "dGVybTo1Mw==",
            "name": "Another Menu",
            "databaseId": 53,
            "menuItems": {
              "nodes": [
                {
                  "id": "bmF2X21lbnVfaXRlbTo2MzQ=",
                  "databaseId": 634,
                  "menu": {
                    "node": {
                      "name": "Another Menu"
                    }
                  }
                },
                {
                  "id": "bmF2X21lbnVfaXRlbTo2MzU=",
                  "databaseId": 635,
                  "menu": {
                    "node": {
                      "name": "Another Menu"
                    }
                  }
                },
                {
                  "id": "bmF2X21lbnVfaXRlbTo2MzY=",
                  "databaseId": 636,
                  "menu": {
                    "node": {
                      "name": "Another Menu"
                    }
                  }
                }
              ]
            }
          }
        }
      ]
    }
  }
}
```

----

In Action:

![Screen Shot 2020-06-24 at 10 29 49 PM](https://user-images.githubusercontent.com/1260765/85653408-41814180-b66a-11ea-9b53-eaa584564606.png)
